package.json

```diff
@@ -2,7 +2,8 @@
   "private": true,
   "scripts": {
     "build": "turbo run build",
-    "dev": "turbo run dev",
+    "migrate": "cd packages/db && pnpm migrate",
+    "dev": "pnpm migrate && turbo run dev",
```

Comment:
I am not sure if we should run it inside of the root `package.json`. It feels to be more appropriate to run inside of `/apps/web`, wdyt?

Reply:
You're spot on with that. I think I was thinking in another direction but things are much clearer now. 

I guess we can go ahead and close this.

Reply:
I'll close this one for now!